src/main/java/teammates/common/datatransfer/FeedbackQuestionAttributes.java

```diff
@@ -565,6 +585,15 @@ public Text getQuestionMetaData() {
         return questionMetaData;
     }
 
+    public Text getQuestionDescription() {
+        return questionDescription == null ? null
+                : new Text(Sanitizer.sanitizeForRichText(questionDescription.getValue()));
```

Comment:
If the description is sanitized on the constructor, I don't think this is needed anymore. Just a simple `return questionDescription` will do.